src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/types/global.ts

```diff
@@ -130,7 +131,7 @@ export interface IFormInputComponent extends IFormComponent {
 }
 
 export interface IFormRadioButtonComponent extends IFormComponent {
-  options: IOptions[];
+  options: IOption[];
```

Comment:
Could one use IOptions here?

Reply:
No. This is an array of IOption, and not a key-value structure as IOptions.